R/demog_pyears.R

```diff
@@ -73,6 +74,30 @@ demog_pyears <- function(formula, data, period=NULL, agegr=NULL, cohort=NULL, ti
   })
   aggr <- Reduce(rbind, aggr_spl)
 
+  if (nrow(aggr) == 0) {
```

Comment:
have you compared this with some of the changes Haley made recently to fill missing values outside of these functions in the survey script? Seems like we should choose one method or the other to fill values. @hcomfo95